src/track/taglib/trackmetadata_id3v2.cpp

```diff
@@ -382,10 +382,10 @@ void writeTextIdentificationFrame(
         auto pFrame =
                 std::make_unique<TagLib::ID3v2::TextIdentificationFrame>(id, stringType);
         pFrame->setText(toTString(text));
-        pTag->addFrame(pFrame.get());
-        // Now that the plain pointer in pFrame is owned and managed by
-        // pTag we need to release the ownership to avoid double deletion!
-        pFrame.release();
+
+        // Release returns a pointer to the managed object and releases the ownership,
+        // the plain pointer in pFrame is owned and managed by pTag
```

Comment:
Something like this should be sufficient 
```suggestion
        // pTag takes the ownership of pFrame 
```